corehq/apps/userreports/static/userreports/js/configure_report.js

```diff
@@ -0,0 +1,40 @@
+/* global reportBuilder */
+hqDefine('userreports/js/reportbuilder/configure_report', function() {
+    var initialPageData = hqImport('hqwebapp/js/initial_page_data');
+
+    window.ReportBuilder = {
```

Comment:
Adding this `window` is a technically safe/reasonable thing to do, but it doesn't deal with the underlying problem that `ReportBuilder` is a global. It doesn't look like a widely used one: grepping for ReportBuilder on the front end doesn't turn up much. You may be able to just replace references to `ReportBuilder.Constants` with `hqImport("userreports/js/constants")`.